### DISCUSSION TITLE
I don't understand spawn point cords.

### DISCUSSION THREAD

**Discussion by Civitello** [Zodsmar/SeedSearcherStandaloneTool#123] [Q&A]
What are the coordinates on the spawn point option for?
Also, I have read in these forums that there are tool tips, I don't seem to be seeing them anywhere.

**Comment by Zodsmar** [comment]
Yeah the whole spawnpoint system is really weird. It doesnt do what you think it would. You essentially give an X and Z coord and say you want seeds with that specific spawn point. The new version will be not that. The new version will be for toggling if you want to search around spawnpoint or 0, 0 (So origin) Essentially you can toggle between 0, 0 and making the spawnpoint your new "origin"
